config.yml

```diff
@@ -216,10 +216,10 @@ filecoin:
   api: https://api.filscan.io:8700/rpc/v1
   explorer: https://filfox.info
 
-# smartchain:
-#   api: <blockbook api>
-#   rpc: https://bsc-dataseed.nariox.org
-#   collections_api: https://nftview.bounce.finance
+smartchain:
+  api: <blockbook api>
+  rpc: https://bsc-dataseed.binance.org
```

Comment:
`smartchain.rpc` is this still required url? I don't think we ever use it in the blockatlas, same for ethereum

Reply:
ok, let me remove it

Reply:
done